src/plugins/kibana/public/management/app.html

```diff
@@ -10,9 +10,11 @@
       <li
         ng-if="sectionName"
         ng-repeat="item in section.items.inOrder"
-        ng-class="item.class">
+        ng-class="item.class"
+        tooltip="{{::item.tooltip}}" tooltip-placement="bottom" tooltip-popup-delay="400" tooltip-append-to-body="1"
+      >
 
-        <a class="navbar-link" kbn-href="{{::item.url}}" data-test-subj="{{::item.name}}">
+        <a class="navbar-link" ng-class="{ 'is-navbar-nav-link-disabled': !item.path }" kbn-href="{{::item.url}}" data-test-subj="{{::item.name}}">
```

Comment:
That doesn't work because when neither `path` nor `url` are set, the `url` is being set to '#/management/undefined', presumably by [this line of code](https://github.com/elastic/kibana/blob/master/src/ui/public/management/section.js#L26).